workload/terraform/greenfield/ADDSscenario/provider.tf

```diff
@@ -14,13 +16,18 @@ terraform {
     }
     azapi = {
       source = "Azure/azapi"
+      version = "=1.8.0"
     }
     time = {
       source = "hashicorp/time"
     }
   }
 }
 
+provider "azuread" {
+  tenant_id = "b7b812fe-be8c-4adc-883f-2a3c36d753d7"
```

Comment:
The tenant_id needs to be removed, it is not needed here.

Reply:
Removed